src/signals/settings/roles/containers/RoleFormContainer/components/RoleForm/__test__/RoleForm.test.js

```diff
@@ -84,9 +77,28 @@ describe('/signals/settings/roles/components/RoleForm', () => {
     };
 
     expect(props.onPatchRole).not.toHaveBeenCalled();
-
     fireEvent.change(getByTestId('rolesFormFieldName'), event);
 
+    act(() => {
+      fireEvent.click(container.querySelectorAll('input[type="checkbox"]')[0]);
+    });
+
+    await findByTestId('rolesFormFieldName');
+
+    fireEvent.click(getByTestId('submitBtn'), { preventDefault: jest.fn() });
+
+    expect(props.onPatchRole).toHaveBeenCalledWith({
+      id: 2,
+      name: 'Behandelaar',
+      permission_ids: [permissionsJson[0].id, 110, 164, 162, 163, 161, 111],
```

Comment:
Please refrain from using hardcoded values. Instead, use references to props in the fixture. This prevents the test from breaking in the future when the fixture data is replaced or updated.

Reply:
done